src/renderer/components/RouterComponents/ProjectsList/IProject.ts

```diff
@@ -4,8 +4,9 @@ export interface IProject {
   id: number;
   image: string;
   name: string;
-  date_creation: Date;
-  date_edition: Date;
+  // Dates have to be serializable, cannot be of type Date
```

Comment:
@Eleara because of this change, the sorting in ProjectsList does not work properly. It may be enough to simply do `new Date(date_creation)` before comparing these dates, but I did not want to mess up your algorithm.